spec/person_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './person'
+
+describe Person do
+  before :each do
+    @person = Person.new 20, 'Roland'
+    @person_unknown = Person.new 33
+  end
+
+  it 'should returns a Person object' do
+    expect(@person).to be_instance_of Person
+    expect(@person_unknown).to be_instance_of Person
+  end
+
+  it 'should show the person age' do
+    expect(@person.age).to eq 20
+    expect(@person_unknown.age).to eq 33
+  end
+
+  it 'should show the person name' do
+    expect(@person.name).to eq 'Roland'
+  end
+
+  it 'should show the person name as Unknown' do
+    expect(@person_unknown.name).to eq 'Unknown'
+  end
+
+  it 'should return the person name' do
+    @person.correct_name
+    expect(@person.name).to eq 'Roland'
+  end
+
+  it 'should return true if person age is less than 18 otherwise return true' do
+    expect(@person.can_use_services?).to be false
+  end
+end
```

Comment:
- [ ] Great job with test coverage so far, but since we are aiming for a 100% test coverage, it will be nice for us to cover all methods in all our classes with tests. For example in this `Person class`, we could add the following tests: 
- Test for adding a new rental , something like so:
```
  context "When adding a new rentals" do

    it "The add_rental method returns a rental and adds its to the book" do
       --- Fill in the content --- 
    end
  end
```
- Test for checking permissions, something like so: 
```ruby
  context "When checking permissions" do
    
    # *check theres some positive and negative case scenarios
    # *can_use_services? negative scenario
    it "The can_use_services? method returns false if under age and no parent permission" do
      --- Fill in the content ---
    end
    
    # *can_use_services? positive scenario
    it "The can_use_services? method returns true if over age" do
     --- Fill in the content --- 
    end
    
    it "The can_use_services? method returns true if has parent permission" do
      --- Fill in the content --- 
    end
  end
```
- Test for validating name, some like so: 
```ruby
  context "When validating name" do

    it "The validate_name method corrects the name" do
      --- Fill in the content --- 
    end
  end
```
#### Please let make all our methods in all our classes are covered with tests.